pkg/console/console.go

```diff
@@ -62,6 +64,8 @@ func Pull(apiClient *clients.Settings, name string) (*Builder, error) {
 		glog.V(100).Info("The name of the Console is empty")
 
 		builder.errorMsg = "console 'name' cannot be empty"
+
+		return builder, fmt.Errorf("error: %s", builder.errorMsg)
```

Comment:
```suggestion
		return nil, fmt.Errorf("console 'name' cannot be empty")
```
since this is just in Pull I don't think we need builder.errorMsg